### DISCUSSION TITLE
Actions down for anyone?

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#110222] [Actions] [answered]
### Select Topic Area

Question

### Body

.

**Comment by ettaboyle** [comment, marked as answer]
Hi @adamlui,

Thanks for sharing and we appreciate your patience. You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

Current Status: Investigating reports of degraded performance (CHECK THE EXACT REASON FOR THE OUTAGE AND REPLACE THIS IF NEEDED)
